test/util/report.js

```diff
@@ -92,14 +89,13 @@ const getAttributeRowName = function ($, $tableRow) {
  * Processes all example files to find data-test-ids and missing data-test-ids
  * Builds exampleCoverage object:
  * {
- *   <exampleFile>: {
- *     existingTestIds: <array of discovered data-test-ids>
- *     missingTests: <copy of array of discovered data-test-ids>
- *     missingAttrs: <rows in attribute table missing data-test-ids>
- *     missingKeys: <rows in keyboard support table missing data-test-ids>
- *   }
+ * <exampleFile>: {
+ * existingTestIds: <array of discovered data-test-ids>
+ * missingTests: <copy of array of discovered data-test-ids>
+ * missingAttrs: <rows in attribute table missing data-test-ids>
+ * missingKeys: <rows in keyboard support table missing data-test-ids>
+ * }
```

Comment:
I think we want to allow contributors to preserve their own formatting in JSDoc for instances like this, so readability could be clearer. So disabling `jsdoc/check-line-alignment` should be fine.